xml/en/docs/njs/reference.xml

```diff
@@ -77,8 +77,8 @@ List of all njs properties and methods can be found in
 <para>
 The HTTP request object is available only in the
 <link doc="../http/ngx_http_js_module.xml">ngx_http_js_module</link> module.
-All string properties of the object are
-<link id="string">byte strings</link>.
+All string properties of the object before <link doc="changes.xml" id="njs0.8.5">0.8.5</link>
```

Comment:
"Before" should be in the beginning or end (for consistency in the beginning)
Before <link doc="changes.xml" id="njs0.8.5">0.8.5</link>, all string properties ... 
or 
All string properties were byte strings before 0.8.5.